src/harness/fake_sas.py

```diff
@@ -209,6 +209,10 @@ def do_POST(self):
                        '/admin/injectdata/cluster_list',
                        '/admin/injectdata/pal_database_record'):
```

Comment:
I think you need to add "esc_reset" to this list and "esc_zone" in its own 'elif' below. I thought they were in your earlier code; did you take them out? It looks like InjectEscZone() also disappeared. Am I missing something?